src/abhijitMJun20/CountLetters.java

```diff
@@ -0,0 +1,31 @@
+package abhijitMJun20;
+
+public class CountLetters {
+	void charfind(String input)
+	{ 
+		 int count=0,count1=0,count2=0,count3=0;
```

Comment:
First thing I really don't like the way you have given variable name. Give a proper name.
Like countLowercase...etc. Update it after I will check the rest of the code.